lib/src/transformer/impl_generation.dart

```diff
@@ -409,6 +409,31 @@ class ImplGenerator {
     Map keyConstants = {};
     Map constants = {};
 
+    typedMap.node.members
+        .where((member) => member is FieldDeclaration && member.isStatic)
+        .forEach((_field) {
+          final field = _field as FieldDeclaration; // ignore: avoid_as
+
+          annotations.Accessor accessorMeta = instantiateAnnotation(field, annotations.Accessor);
```

Comment:
Can we skip lines 417-425? I don't think there should ever be an instance where a `doNotGenerate()` annotation is added to a `meta` field